### PR TITLE
Pandaria Empire

### PR DESCRIPTION
**Pull request by zumbak04**
## Developer changelog:
- All the Pandaria cultures except Mantid can restore the Pandaria Empire (name depends on culture) via the decisions.

# How to test:
Start as the Xin'ze Kingdom in any bookmark.

`usurp e_kun_lai` to get an emperor title, use `yesman`. Now you're able to offer vassalization to everyone around you and they will agree. Thus you will get all the needed duchies.

Also, use `prestige 10000` to get 10000 prestige.

Use these commands to changed cultures when you're emperor:
- `culture mogu`
- `culture jinyu`
- `culture mainland_pandaren`
- `culture jungle_hozen`

### CODE REVIEW

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -3057,6 +3057,14 @@ RULE_MISTS_OF_PANDARIA_ON;On;;;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_ON_DESC;The Mists of Pandaria are enabled, making Pandaria isolated till they're gone around the 613th year.;;;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_OFF;Off;;;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_OFF_DESC;The Mists of Pandaria are disabled thus Pandaria isn't isolated.;;;;;;;;;;;;;x
+emperor_of_pandaria;Blood of [This.GetFirstNameWithNick];;;;;;;;;;;;;x
+emperor_of_pandaria_desc;The next emperor of Pandaria after legendary Shaohao, [This.GetFirstNameWithNick] restored imperial traditions and united Pandaria once again.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3061]:
```suggestion
emperor_of_pandaria_desc;The next emperor of Pandaria after the legendary Shaohao, [This.GetFirstNameWithNick] restored the imperial traditions and united Pandaria once again.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 2]
```diff
@@ -3057,6 +3057,14 @@ RULE_MISTS_OF_PANDARIA_ON;On;;;;;;;;;;;;;x
...
+form_the_pandaria_empire;Restore the Pandaria Empire;;;;;;;;;;;;;x
+form_the_pandaria_empire_desc;You become the next emperor of Pandaria after legendary Shaohao.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3063]:
```suggestion
form_the_pandaria_empire_desc;You will become the next emperor of Pandaria after the legendary emperor Shaohao.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 3]
```diff
@@ -3057,6 +3057,14 @@ RULE_MISTS_OF_PANDARIA_ON;On;;;;;;;;;;;;;x
...
+EVTTITLE_WCPAN_200;[e_pandaria_empire.GetName];;;;;;;;;;;;;x
+EVTDESC_WCPAN_200;Ten thousand years ago, the last emperor of Pandaria, Shaohao, became one with the land and disappeared. His last breath became the mist that veiled Pandaria for ages. After the emperor disappeared, the power in Pandaria shifted to the local level. With all threats walled off, there was no longer a need for an emperor anymore... but times change.\n\n[target_emperor.GetTitledFirstName] belives that Pandaria needs the emperor�s strong hand once again. Controlling most of Pandaria, [target_emperor.GetSheHe] declares [target_emperor.GetHerselfHimself] the next emperor after Shaohao.;;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3065]:
```suggestion
EVTDESC_WCPAN_200;Ten thousand years ago, the last emperor of Pandaria, Shaohao, became one with the land and disappeared. His last breath became the mist that veiled Pandaria for ages. After the emperor disappeared, power in Pandaria shifted to the local level. With all threats walled off, there was no longer a need for an emperor anymore... but times have changed.\n\n[target_emperor.GetTitledFirstName] believes that Pandaria needs the emperor�s strong hand once again. Controlling most of Pandaria, [target_emperor.GetSheHe] declares [target_emperor.GetHerselfHimself] the next emperor after Shaohao.;;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 4]
```diff
@@ -3057,6 +3057,14 @@ RULE_MISTS_OF_PANDARIA_ON;On;;;;;;;;;;;;;x
...
+EVTOPTA_WCPAN_200;I'm the hearth of Pandaria!;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3066]: Just my opinion, but would `heart` not be better here than `hearth`?
```suggestion
EVTOPTA_WCPAN_200;I'm the heart of Pandaria!;;;;;;;;;;;;;x
```

Comment (zumbak04) [reply localisation/00_zuwc_united_localisation.csv]: Done in c1b4853ae2a70fe922a78be01fa3cbe020564096.

File: events/wc_pandaria_events.txt [hunk 5]
```diff
@@ -75,4 +75,58 @@ narrative_event = {
...
+narrative_event = {
+	id = WCPAN.200
+	title = EVTTITLE_WCPAN_200
+	desc = EVTDESC_WCPAN_200
+	picture = GFX_evt_pandaria_twin_monoliths
+	border = GFX_event_narrative_frame_diplomacy
+
+	is_triggered_only = yes
+
+	# I'm the hearth of Pandaria!
```
Comment (Myrran) [inline events/wc_pandaria_events.txt:91]: # I am the heart of Pandaria!

File: localisation/00_zuwc_united_localisation.csv [hunk 6]
```diff
@@ -3095,6 +3095,14 @@ RULE_MISTS_OF_PANDARIA_ON;On;;Aktiviert;;;;;;;;;;;x
...
+emperor_of_pandaria_desc;The next emperor of Pandaria after the legendary Shaohao, [This.GetFirstNameWithNick] restored the imperial traditions and united Pandaria once again.;;;;;;;;;;;;;x
+form_the_pandaria_empire;Restore the Pandaria Empire;;;;;;;;;;;;;x
+form_the_pandaria_empire_desc;You will become the next emperor of Pandaria after the legendary emperor Shaohao.;;;;;;;;;;;;;x
+EVTTITLE_WCPAN_200;[e_pandaria_empire.GetName];;;;;;;;;;;;;x
+EVTDESC_WCPAN_200;Ten thousand years ago, the last emperor of Pandaria, Shaohao, became one with the land and disappeared. His last breath became the mist that veiled Pandaria for ages. After the emperor disappeared, power in Pandaria shifted to the local level. With all threats walled off, there was no longer a need for an emperor anymore... but times have changed.\n\n[target_emperor.GetTitledFirstName] believes that Pandaria needs the emperor's strong hand once again. Controlling most of Pandaria, [target_emperor.GetSheHe] declares [target_emperor.GetHerselfHimself] the next emperor after Shaohao.;;;;;;;;;;;;;x
+EVTOPTA_WCPAN_200;I'm the heart of Pandaria!;;;;;;;;;;;;;x
```
Comment (Myrran) [inline localisation/00_zuwc_united_localisation.csv:3104]: EVTOPTA_WCPAN_200;I am the heart of Pandaria!;;;;;;;;;;;;;x

File: localisation/00_zuwc_united_localisation.csv [hunk 7]
```diff
@@ -3095,6 +3095,14 @@ RULE_MISTS_OF_PANDARIA_ON;On;;Aktiviert;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_ON_DESC;The Mists of Pandaria are enabled, making Pandaria isolated until they part around the 613th year. ;;Die Nebel von Pandaria sind aktiviert, Interaktionen sind bis zum Jahr 613 blockiert.;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_OFF;Off;;Deaktiviert;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_OFF_DESC;The Mists of Pandaria are disabled, making Pandaria open to the world. ;;Die Nebel von Pandaria sind deaktiviert, Pandaria steht der Welt offen.;;;;;;;;;;;x
+emperor_of_pandaria;Blood of [This.GetFirstNameWithNick];;;;;;;;;;;;;x
+emperor_of_pandaria_desc;The next emperor of Pandaria after the legendary Shaohao, [This.GetFirstNameWithNick] restored the imperial traditions and united Pandaria once again.;;;;;;;;;;;;;x
+form_the_pandaria_empire;Restore the Pandaria Empire;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3100]: I've heard it called the Pandaren Empire, but not the Pandaria Empire. I feel like the Empire of Pandaria makes more sense in this context, as it is the empire ruling the land of Pandaria.
```suggestion
form_the_pandaria_empire;Restore the Empire of Pandaria;;;;;;;;;;;;;x
```

File: localisation/000_Other_title_names.csv [hunk 8]
```diff
@@ -461,3 +461,9 @@ c_south_islands;Jaguero;;Jaguero;;;;;;;;;;;x
 c_forlorn_3;Bloodsail;;Blutsegel;;;;;;;;;;;x
 c_south_islands_adj;Jaguero;;jaguerisch;;;;;;;;;;;x
 c_forlorn_3_adj;Bloodsail;;blutseglisch;;;;;;;;;;;x
+e_pandaria_empire;Pandaria Empire;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/000_Other_title_names.csv:464]:
```suggestion
e_pandaria_empire;Empire of Pandaria;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv (hunk 7 above)
Comment (zumbak04) [reply localisation/00_zuwc_united_localisation.csv:3100]: Done in 51a4ac3c22c01f0b263199af3ba03e1d17d4b91c.

**Comment by zumbak04** [comment]
> I did want to make a suggestion though. Its a bit odd for Yaungol or Mogu to reference Shaohao so reverently, and to a degree, the same is true of the Jinyu and Hozen. Would it be possible for there to be culture specific text upon the declaration of the empire?

More like one neutral, but I have no idea what to write there. Every nation had an empire at some point except the yaungol and sauroks thus I've seen it as different shades of one empire starting with the mogu.

**Comment by LordSilvermane** [comment]
@zumbak04 Fair enough. I could try to write up a generic neutral text for the non-pandarian races if you want. Otherwise looks good to me.

File: localisation/000_Other_title_names.csv [hunk 9]
```diff
@@ -461,3 +461,9 @@ c_south_islands;Jaguero;;Jaguero;;;;;;;;;;;x
 c_forlorn_3;Bloodsail;;Blutsegel;;;;;;;;;;;x
 c_south_islands_adj;Jaguero;;jaguerisch;;;;;;;;;;;x
 c_forlorn_3_adj;Bloodsail;;blutseglisch;;;;;;;;;;;x
+e_pandaria_empire;Pandaria;;;;;;;;;;;;;x
+e_pandaria_empire_adj;Pandaria;;;;;;;;;;;;;x
```
Comment (arithon) [inline localisation/000_Other_title_names.csv]: shouldn't this be Pandarian or Pandaren?

**Pull request merged**